services/hackernews/hackernews-user-karma.tester.js

```diff
@@ -0,0 +1,16 @@
+import { createServiceTester } from '../tester.js'
+import { isMetric } from '../test-validators.js'
+
+export const t = await createServiceTester()
+
+t.create('valid repo -- live').get('/pg.json').expectBadge({
+  label: 'pg',
+  message: isMetric,
+})
+
+t.create('invalid user -- live')
```

Comment:
We don't need to add "-- live" to the tests. The service test runner will automatically prefix tests with `[live]` or `[mocked]` as applicable.